plugins/egis-moc/fu-egis-moc-device.c

```diff
@@ -330,6 +330,9 @@ fu_egis_moc_device_setup(FuDevice *device, GError **error)
 		}
 	}
 
+	/* delay for 500ms for usb setup */
+	fu_device_sleep(device, 500);
```

Comment:
This is going to increase the startup time by about 30% on tens of thousands of machines -- what bug are you trying to work around?